redash/query_runner/impala_ds.py

```diff
@@ -82,11 +82,11 @@ def __init__(self, configuration):
     def _get_tables(self, schema_dict):
         schemas_query = "show schemas;"
         tables_query = "show tables in %s;"
-        columns_query = "show column stats %s;"
+        columns_query = "show column stats %s.%s;"
 
-        for schema_name in map(lambda a: a['name'], self._run_query_internal(schemas_query)):
-            for table_name in map(lambda a: a['name'], self._run_query_internal(tables_query % schema_name)):
-                columns = map(lambda a: a['Column'], self._run_query_internal(columns_query % table_name))
+        for schema_name in filter(lambda a: len(a) > 0, map(lambda a: str(a['name']), self._run_query_internal(schemas_query))):
```

Comment:
Thank you!

Since there is no character other than ASCII , Change ```str``` to ```unicode```

>Also, in what cases it will be null/ of length 0?

I'm sorry... , table scheme is length 0 scheme no case